mmf/common/test_reporter.py

```diff
@@ -159,9 +159,12 @@ def add_to_report(self, report, model):
             if "question_id" in report:
                 report.question_id = gather_tensor(report.question_id).view(-1)
             if "image_id" in report:
-                _, enc_size = report.image_id.size()
                 report.image_id = gather_tensor(report.image_id)
-                report.image_id = report.image_id.view(-1, enc_size)
+                if len(report.image_id.shape) == 2:
```

Comment:
One should check the shape before (not after) `gather_tensor` (fixed in https://github.com/facebookresearch/mmf/pull/324)